src/Microsoft.TemplateEngine.Edge/Settings/InstallationScope.cs

```diff
@@ -15,6 +15,7 @@ public enum InstallationScope
         /// </summary>
         Global = 0,
 
+#pragma warning disable CS1587 // XML comment is not placed on a valid language element
```

Comment:
I wonder if we should get rid of the commented code. If it is needed in the future, we can add it then.

Reply:
I would prefer to keep it, as likely we will do it soon.  Otherwise having enum with one value is not logical.